sn_client/src/files/upload.rs

```diff
@@ -461,7 +464,8 @@ impl FilesUpload {
                 .await
             {
                 Ok(cost) => {
-                    debug!("Storecosts retrieved for {:?} {cost:?}", chunk_info.name);
+                    let t = &cost.quote;
+                    debug!("Storecosts retrieved for {:?} {t:?}", chunk_info.name);
```

Comment:
Please check over this, I do not know the original values of the message but I gave it an estimate.

Reply:
why is `t` `cost.quote`? what is `t`? can we call this `quote`?

Reply:
t is for temp until quote is confirmed. I can rename it now👍